JitHub.AuthFunction/GithubAuth.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Octokit;
+using System.Text.Json;
```

Comment:
Where is this json serialization/deserialization used? (or could any of the using statements  be cleared?